### PR TITLE
Hash-slinger: init at  2.7.0

### PR DESCRIPTION
**Pull request by leenaars**
###### Motivation for this change

Hash-slinger is a very useful tool by Paul Wouters et al to create special DNS records. 
The Unbound Python module is a dependency which may be of use itself, but it was not yet available - unlike Unbound/libunbound itself.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/networking/unbound/python.nix [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchurl, openssl, expat, libevent, swig, python, pythonPackages }:
```
Comment (FRidh) [inline pkgs/tools/networking/unbound/python.nix]: python is an attribute of pythonPackages, so no need to add both here. Having only pythonPackages will make it easier and less error-prone when using a custom python.

**Comment by FRidh** [comment]
`py-unbound` seems to be an extension module; it is a package used as a library. In that case it needs to go via `python-packages.nix`.

File: pkgs/tools/networking/unbound/python.nix (hunk 1 above)
Comment (leenaars) [inline pkgs/tools/networking/unbound/python.nix]: @FRidh removed python. Thanks for the tip! Anything else?

**Comment by leenaars** [comment]
@FRidh: python was referenced as a function as well in the installPhase, which I just repaired. Instead of 
${python.sitePackages} 
it should be:
${pythonPackages.python.sitePackages}/

**Comment by FRidh** [comment]
@leenaars interesting, github shows a different state than my local checkout...

**Comment by FRidh** [comment]
okay, now its correct again. 

I see, I'll push a fix.

**Pull request merged**